### ISSUE TITLE
DPL-567 Bioscan: Insect Lysate Preparation pipeline configuration

### ISSUE THREAD

**Issue by andrewsparkes** [sanger/limber#1195]
**User story**
Add new pipeline, purpose, and QC file export configs for Lysate prep. Capture in purpose config the well exclusion locations where the controls cannot be placed and where the positive control will be. The Labware creator for the Lysate plate (separate story) needs to randomise the negative control location and place those samples. The robot bed verifications (separate stories) check the automation setup.

**Who are the primary contacts for this story**
Scott T
Emma B

**Who is the nominated tester for UAT**
TBC

**Acceptance criteria**
To be considered successful the solution must allow:
- [x] use of insect lysate prep pipeline via Limber
- [x] insect lysate prep integration test should run

**Dependencies**
This story is blocked by the following dependencies:
NA

References
This story has a non-blocking relationship with:
Integration test - https://github.com/sanger/General-Backlog-Items/issues/233
Limber pipeline configuration - https://github.com/sanger/sequencescape/issues/3703
Labware creator for Lysate plate - https://github.com/sanger/limber/issues/1196
Limber bed verification for Stock plate location - https://github.com/sanger/limber/issues/1198
Limber bed verification for Stock to Lysate plate - https://github.com/sanger/limber/issues/1199

**Additional context**
This is the Limber part of the pipeline configuration.
See diagram [here](https://lucid.app/lucidchart/b21a6020-df09-4be1-af69-26403ed467a1/edit?viewport_loc=1860%2C1619%2C1885%2C1065%2Cwi47cHrbc6ZW&invitationId=inv_44a43cc4-479b-4e19-924c-2a0182f4255c)

**Comment by harrietc52** [comment]
Removing story from board as referencing story in EPIC issue https://github.com/sanger/limber/issues/1270
